example/support/utree/error_handler.hpp

```diff
@@ -74,9 +74,9 @@ struct expected_component : std::exception
         msg = oss.str();
     }
 
-    virtual ~expected_component() throw() {}
+    virtual ~expected_component() noexcept {}
```

Comment:
I thought that things in "example" would use standard C++ rather than boost specific macros (as most people are using C++11 nowadays). But as you wish.

Reply:
Done.

Reply:
The examples are part of CI (I did not add them to builds just because we already hitting time limits, but it is on my TODO list).